q2_anomaly_detection/models.py

```diff
@@ -0,0 +1,75 @@
+import numpy as np
+
+class Cloud():
+    def __init__(self, n_neighbors="auto", metric="precomputed", percentile_cutoff=0.95):
+        """
+        n_neighbors: Interger/String default='auto'
+            'auto' will set k = 10% of reference set size
+        """
+        self.k = n_neighbors
+        self.metric=metric
+        self.percentile_cutoff = percentile_cutoff
+        
+    def fit(self, X):
```

Comment:
It is idiomatic of sklearn to have a `y=None` argument, even if it is unused. This can be helpful for pipe-lining.
```suggestion
    def fit(self, X, y=None):
```